Zappy_IA_Report/NN/Code/Tutorial1.py

```
@@ -0,0 +1,18 @@
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import matplotlib.pyplot as plt
+
+data = keras.datasets.fashion_mnist
+(train_images, train_labels), (test_images, test_labels) = data.load_data()
+
+class_names = ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat',
+        'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)